scripts/pushArgos.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-console */
```

Comment:
Please move this to the appropriate lines. We have these rules for a reason.

Reply:
What about we disable the no-console rule for the scripts? The motivation for this rule is so we don't ship code with `console.log('wtf')`.

Reply:
We disable eslint console in the header of most of the other scripts. Seems like a pattern.

Reply:
> We disable eslint console in the header of most of the other scripts. 

Ok? These are incorrect as well so we shouldn't proliferate the problematic pattern.

Reply:
I mean, what's the advantage of this eslint rule for the script folder?